### ISSUE TITLE
Publish on JCenter/Maven Central

### ISSUE THREAD

**Issue by rock3r** [cyrilmottier/QueryHighlighter#1]
This code and the accompanying article are very interesting, thanks @cyrilmottier! I am just in the process of implementing search in a few apps and this saves me quite some time :) I was wondering, would it be possible to have a release on JCenter and/or Maven Central so I can use it as a remote dependency instead of dropping the code?

**Comment by cyrilmottier** [comment]
QueryHighlighter is more of a "here's a way to do it" rather than a "here's what you should add to your dependencies". Indeed, the API works great in some particular cases but might not suit your needs if, for instance, the highlight part is determined by an external entity.

As a consequence, I don't think it would worth putting it on JCenter/Maven.

**Comment by rock3r** [comment]
Makes sense, I will probably just use it as a reference/baseline and build on it. Thanks again for the great work!

**Comment by shivasurya** [comment]
@cyrilmottier i followed the repo but set on Item Click listener for listview ? can you please help me in this issue.
